includes/class-wc-payments-account.php

```diff
@@ -1499,9 +1499,12 @@ private function init_stripe_onboarding( $wcpay_connect_from, $additional_args =
 			$account_data = [];
 		}
 
+		$source = sanitize_text_field( wp_unslash( $_GET['source'] ) );
+
 		$site_data = [
 			'site_username' => wp_get_current_user()->user_login,
 			'site_locale'   => get_locale(),
+			'source'        => $source,
```

Comment:
I think data point is better suited under user_data since it is about the user-initiated onboarding origin.

Reply:
Addressed in https://github.com/Automattic/woocommerce-payments/pull/8668/commits/6805615ab089eb023e57737d80ae7d96248ed2c4